tests/test_span_session.rb

```diff
@@ -0,0 +1,99 @@
+require_relative 'ciscotest'
+require_relative '../lib/cisco_node_utils/span_session'
+require_relative '../lib/cisco_node_utils/interface'
+require_relative '../lib/cisco_node_utils/vlan'
+require_relative '../lib/cisco_node_utils/cisco_cmn_utils'
+
+include Cisco
+
+# TestSpanSession - Minitest for SPAN session node utility
+class TestSpanSession < CiscoTestCase
+  @skip_unless_supported = 'session'
+  @@cleaned = false # rubocop:disable Style/ClassVars
+
+  def setup
+    super
+    cleanup unless @@cleaned
+    @@cleaned = true # rubocop:disable Style/ClassVars
+  end
+
+  def teardown
+    cleanup
+  end
+
+  def cleanup
+    SpanSession.sessions.each do |_session, obj|
+      obj.destroy
+    end
+  end
+
+  def test_create_new_session
+    span = SpanSession.new(1)
+
+    assert_equal(span.session_id, 1)
+    assert_equal(span.type, 'local') # default session type
+    assert(span.shutdown)
+    span.destroy
+  end
+
+  def test_remove_session
+    span = SpanSession.new(1)
+    span.destroy
+    refute(span, 'Session was not cleaned up correctly...')
+  end
+
+  def test_create_session_invalid_id
+    e = assert_raises(CliError) { SpanSession.new(33) }
+    assert_match(/Invalid value.range/, e.message)
+  end
+
+  def test_session_type
+    span = SpanSession.new(1)
+    erspan_type = 'erspan-source'
+    span.type = erspan_type
+    assert_equal(span.type, erspan_type)
+    span.destroy
+  end
+
+  def test_session_description
+    span = SpanSession.new(1)
+    desc = 'SPAN session 1'
+    span.description = desc
+    assert_equal(span.description, desc)
+    span.destroy
+  end
+
+  def test_session_source_interface
+    span = SpanSession.new(1)
+    po_int = Interface.new('port-channel1')
+    ints = { 'Ethernet1/1' => 'rx',
```

Comment:
The rubocop errors are caused by this section of code:

```
Offenses:

lib/cisco_node_utils/span_session.rb:5:1: C: Extra blank line detected.

tests/test_span_session.rb:69:14: C: Align the elements of a hash literal if they span more than one line.

    ints = { 'Ethernet1/1' => 'rx',

             ^^^^^^^^^^^^^^^^^^^^^

tests/test_span_session.rb:70:14: C: Align the elements of a hash literal if they span more than one line.

             'Ethernet1/2' => 'tx',

             ^^^^^^^^^^^^^^^^^^^^^

tests/test_span_session.rb:71:12: C: Align the elements of a hash literal if they span more than one line.

           'port-channel1' => 'both',

           ^^^^^^^^^^^^^^^^^^^^^^^^^

tests/test_span_session.rb:72:17: C: Align the elements of a hash literal if they span more than one line.

                'sup-eth0' => 'rx' }

                ^^^^^^^^^^^^^^^^^^

208 files inspected, 5 offenses detected
```
I believe if you just line them up as follows it will resolve the errors.

```ruby
    ints = { 'Ethernet1/1'   => 'rx',
             'Ethernet1/2'   => 'tx',
             'port-channel1' => 'both',
             'sup-eth0'      => 'rx' }

```